rpm/README.md

```diff
@@ -0,0 +1,35 @@
+# RPM packaging
+
+The template in fingertip.spec contains template for packaging fingertip in
+Fedora and RHEL. As fingertip does not provide any releases yet, the latest
```

Comment:
Actually we already have 0.1.0 and 0.1.1 in pypi. Should I tag those in the repo? 'v0.1.0' or '0.1.0"?

Reply:
Yes, it would be better to have the same versioning everywhere. I did not see them tagged in repo and did not check pypi.

Reply:
Any opinion on preferred tag formatting?

Reply:
I don't have preference. Whatever works for you and whatever works for consistency with pypi. I think just the numbers 0.1.1 should be fine.

Reply:
I kept the packaging of last snapshot so far, but updated to current master. Unless there is something else, I believe it is ready for merge.